src/main/modules/awilix/index.ts

```diff
@@ -1,16 +1,17 @@
-import {asClass, asValue, createContainer, InjectionMode} from 'awilix';
-import {HomeController} from '../../app/controller/HomeController';
-import {Application} from 'express';
-import {CourtsController} from '../../app/controller/courts/CourtsController';
-import {CourtsDownloadController} from '../../app/controller/courts/CourtsDownloadController';
-import {ErrorController} from '../../app/controller/ErrorController';
-import {CourtDetailsController} from '../../app/controller/courts/CourtDetailsController';
-import {BulkUpdateController} from '../../app/controller/bulk-update/BulkUpdateController';
+import { asClass, asValue, createContainer, InjectionMode } from 'awilix';
+import { HomeController } from '../../app/controller/HomeController';
+import { Application } from 'express';
+import { CourtsController } from '../../app/controller/courts/CourtsController';
+import { CourtsDownloadController } from '../../app/controller/courts/CourtsDownloadController';
+import { ErrorController } from '../../app/controller/ErrorController';
+import { CourtDetailsController } from '../../app/controller/courts/CourtDetailsController';
+import { EditCourtController } from '../../app/controller/courts/EditCourtController';
+import { BulkUpdateController } from '../../app/controller/bulk-update/BulkUpdateController';
```

Comment:
Minor issue: the formatting of these imports is different to the rest

Reply:
Removed spaces inline within other imports